scipy/optimize/_differentialevolution.py

```diff
@@ -1048,7 +1052,8 @@ def solve(self):
                                   " solution satisfying the constraints,"
                                   " attempting to polish from the least"
                                   " infeasible solution", UserWarning)
-
+            if self.disp:
+                print("Polishing solution with '%s' method." % polish_method)
```

Comment:
Please make this into an f-string, `print(f"Polishing solution with '{polish_method}'")`